### PR TITLE
Ghc convert export

### PR DESCRIPTION
**Pull request by shayne-fletcher-da**
This PR threads `ModuleEx` throughout the program replacing `Module_`.

### CODE REVIEW

**Comment by shayne-fletcher-da** [comment]
Note : Tests pass, got a bit of linting to take care of.

File: src/HSE/Util.hs [hunk 1]
```diff
@@ -16,7 +16,6 @@ import HSE.Type
 import Data.Functor
 import Prelude
 
-
```
Comment (ndmitchell) [inline src/HSE/Util.hs]: Deleting my beautiful 2 lines after imports...

Comment (shayne-fletcher-da) [reply src/HSE/Util.hs]: Sorry, I will restore. When I finally realized you have a pattern I've tried to respect that.

Comment (shayne-fletcher-da) [reply src/HSE/Util.hs]: ✔️

File: src/Hint/Duplicate.hs [hunk 2]
```diff
@@ -31,8 +31,9 @@ import qualified Data.Map as Map
 duplicateHint :: CrossHint
 duplicateHint ms =
     dupes [(m,d,y) | (m,d,x) <- ds, Do _ y :: Exp S <- universeBi x] ++
+
```
Comment (ndmitchell) [inline src/Hint/Duplicate.hs]: Stray newline?

Comment (shayne-fletcher-da) [reply src/Hint/Duplicate.hs]: ✔️

**Comment by ndmitchell** [comment]
Thanks! An awesome change.

**Pull request merged**